Project setup

arielo2017 opened the pull request:
PR

arielo2017 committed:
Update .gitignore

setup mocha configuration

arielo2017 committed:
Update HelloWord.test.js

setup mocha configuration

aperdomob reviewed (changes requested):
falta el package.json

arielo2017 committed:
Create package.json

setup mocha configuration

aperdomob commented on package.json:
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "test",
```
el nombre del proyecto es psl-workshop-api-testing-js

Commit:
fatal: ambiguous argument 'HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

arielo2017 committed:
Merge branch 'master' into project-setup
